[JIT] Add a python hook for a function to interpret JIT graphs.

ZolotukhinM opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#44493 [JIT] Add a python hook for a function to interpret JIT graphs.**

This function allows to execute a graph exactly as it is, without going
through a graph executor which would run passes on the graph before
interpreting it. I found this feature extremely helpful when I worked on
a stress-testing script to shake out bugs from the TE fuser: I needed to
execute a very specific set of passes on a graph and nothing else, and
then execute exactly it.

Differential Revision: [D23632505](https://our.internmc.facebook.com/intern/diff/D23632505)

Commit:
[JIT] Add a python hook for a function to interpret JIT graphs.

This function allows to execute a graph exactly as it is, without going
through a graph executor which would run passes on the graph before
interpreting it. I found this feature extremely helpful when I worked on
a stress-testing script to shake out bugs from the TE fuser: I needed to
execute a very specific set of passes on a graph and nothing else, and
then execute exactly it.

[ghstack-poisoned]

suo commented on torch/csrc/jit/python/init.cpp:
```diff
@@ -374,6 +374,24 @@ void initJITBindings(PyObject* module) {
             }
             PropagateInputShapes(graph);
           })
+      .def(
```
Can you add a docstring explaining what this is for and what it does?

Commit:
Update on "[JIT] Add a python hook for a function to interpret JIT graphs."


This function allows to execute a graph exactly as it is, without going
through a graph executor which would run passes on the graph before
interpreting it. I found this feature extremely helpful when I worked on
a stress-testing script to shake out bugs from the TE fuser: I needed to
execute a very specific set of passes on a graph and nothing else, and
then execute exactly it.

Differential Revision: [D23632505](https://our.internmc.facebook.com/intern/diff/D23632505)

[ghstack-poisoned]

Commit:
rebase on "[JIT] Add a python hook for a function to interpret JIT graphs."


This function allows to execute a graph exactly as it is, without going
through a graph executor which would run passes on the graph before
interpreting it. I found this feature extremely helpful when I worked on
a stress-testing script to shake out bugs from the TE fuser: I needed to
execute a very specific set of passes on a graph and nothing else, and
then execute exactly it.

Differential Revision: [D23632505](https://our.internmc.facebook.com/intern/diff/D23632505)

[ghstack-poisoned]
